extensions/data-transfer/portability-data-transfer-google/src/main/java/org/dataportabilityproject/datatransfer/google/photos/GooglePhotosExporter.java

```diff
@@ -202,14 +191,12 @@ public GooglePhotosExporter(GoogleCredentialFactory credentialFactory) {
     return new ExportResult<>(resultType, containerResource, continuationData);
   }
 
-  private PicasawebService getOrCreatePhotosService(TokensAndUrlAuthData authData) {
-    return photosService == null ? makePhotosService(authData) : photosService;
+  private GooglePhotosInterface getOrCreatePhotosInterface(TokensAndUrlAuthData authData) {
```

Comment:
This should be synchronized.

Reply:
Volatile?

Reply:
I would just mark the method as synchronized.